### DISCUSSION TITLE
Does AMD GPU support packaging now?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9731] [Storage Provider] [answered]
As a question, do I have an AMD gpu that supports it?

**Comment by Reiers** [comment, marked as answer]
There's no new information about that.
https://github.com/filecoin-project/bellperson/blob/master/README.md#supported--tested-cards

Last I heard was someone who testing on a `5700XT` for windowPost and sealing.
The result was OOM and crashes for windowPost, and for sealing it produces incorrect results and the sectors is never able to pass verification and Prove. 

So as it stands right now, Lotus does not support AMD GPUs.
